spec/forms/serials_form_spec.rb

```diff
@@ -182,7 +181,6 @@
                                        title: [
                                          {
                                            structuredValue: [
-                                             { type: 'odd thing', value: '99' },
```

Comment:
I think it's useful to test that "It doesn't overwrite existing stuff".  Is there a valid type we could use here?

Reply:
Yeah, this change was a mistake. Fixed.